pyiso/caiso.py

```diff
@@ -283,7 +284,24 @@ def get_AS_dataframe(self, anc_region, latest=True, start_at=False, end_at=False
         MW columns holds $/MW float.
         If no data, returns an empty dataframe.
         """
-        # set args
+        # set args, handle latest differently than LMP b/c daily publication schedule
```

Comment:
The problem is that handle-options takes latest=true to mean a 20-minute window.  AS prices are done hourly, so it returns nothing (except maybe if you are calling near the hour).  CAISO publishes the DAM AS prices at 1:00PM the day before the trading day.  This code gets tomorrows AS prices if it is after 1PM and todays if it is before 1PM.  latest=True returns today always also works.  Users can always set start_at / end_at if they want something else